cybozu/config_parser.hpp

```diff
@@ -103,6 +103,23 @@ class config_parser {
         }
     }
 
+    // Get an uint64_t integer converted from the value associated with `key`.
+    // @key   A configuration key.
+    //
+    // Get an uint64_t integer converted from the value associated with `key`.
+    // Raise <not_found> or <illegal_value>.
+    //
+    // @return An uint64_t integer converted from the associated value.
+    std::uint64_t get_as_uint64(const std::string& key) const {
```

Comment:
Need to include `cstdint`, perhaps.
https://en.cppreference.com/w/cpp/header/cstdint

Reply:
Exactly. Fix in https://github.com/cybozu/yrmcds/compare/2211a93f9805f8f2c2180b11848b5df34fb8bc0b..10302cb82c8b0b6a2628c07dbef63beb10a6e232